tests/check_array.c

```diff
@@ -22,17 +22,14 @@ START_TEST(test_take1_ok)
   GpuArray v;
   GpuArray vidx;
   GpuArray vres;
-  static const uint32_t data[24] = { 0,  1,  2,  3,  4,  5,
-                                     6,  7,  8,  9, 10, 11,
-                                    12, 13, 14, 15, 16, 17,
-                                    18, 19, 20, 21, 22, 23};
-  uint32_t buf[12*24];
+  static const uint32_t data[24] = {0,  1,  2,  3,  4,  5,  6,  7,  8,  9,  10, 11,
+                                    12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23};
+  uint32_t buf[12 * 24];
```

Comment:
Yes, lower it to 85.  Also BreakBeforeBraces should be "Attach" (if I read correctly).

And I don't know if there is a way to make it not insert spaces next to binary or (`|`), but that is much clearer.